### PR TITLE
IX Bid Adapter: Outstream Support Update

### PR DESCRIPTION
**Pull request by lksharma**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

This change updates our implementation of the IX outstream video player. 
Key refactors include:
- removal of our hardcoded rendererUrl
- dependency on empty video object
- documentation update

Documentation PR: https://github.com/prebid/prebid.github.io/pull/3777

### CODE REVIEW

**Comment by patmmccann** [comment]
Hi @lksharma could you link you docs repo pull as well

**Comment by lksharma** [comment]
Hey @patmmccann, I'm working on the update to the docs repo and will link it soon. I've changed the PR to draft in the meantime.

**Comment by lksharma** [comment]
Documentation PR: https://github.com/prebid/prebid.github.io/pull/3777

File: modules/ixBidAdapter.js [hunk 1]
```diff
@@ -1128,24 +1128,18 @@ function getCachedErrors() {
...
-
-    // IXOutstreamPlayer supports both vastUrl and vastXml, so we can pass either.
-    // Since vastUrl is going to be deprecated from exchange response, vastXml takes priority.
-    if (bid.vastXml) {
-      window.IXOutstreamPlayer(bid.vastXml, bid.adUnitCode, config);
-    } else {
-      window.IXOutstreamPlayer(bid.vastUrl, bid.adUnitCode, config);
+  bid.renderer.push(function () {
+    const adUnitCode = bid.adUnitCode;
+    const divId = document.getElementById(adUnitCode) ? adUnitCode : getGptSlotInfoForAdUnitCode(adUnitCode).divId;
```
Comment (patmmccann) [inline modules/ixBidAdapter.js:1137]: Generally this all lgtm, my only concern is here, the fallback function might not work as it depends on gpt and matches the first one it finds. You may find you wish to accept configuration parameters here?

**Review by patmmccann — APPROVED** [review]

**Pull request merged**